README.md

```diff
@@ -53,12 +55,13 @@ You are now ready to login to Aras and try out the Call Date Dialog action.
 ## Usage
 
 1. Log in to Aras as admin.
-2. In the main menu, select **Actions > Call Date Dialog**.
+2. From the User menu (upper right corner), select **Actions > Call Date Dialog**.
+   - In v11, use the **Actions** menu to choose the **Call Date Dialog** Action.
 3. When the datepicker dialog appears, choose a date.
 
 The dialog will be dismissed and a browser alert dialog will appear, displaying the selected date value.
 
-![Date Dialog Demo](./Screenshots/demo.gif)
+![Date Dialog Steps](./Screenshots/v12-action_steps.png)
```

Comment:
Let's make this a gif. Name it 12-demo.gif and rename the old one to 11-demo.gif. Keep the old gif in the screenshots folder for reference, but only show the new one in the readme.